.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: test
+on: [push, pull_request]
+jobs:
+  build:
+    name: Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
`windows-latest` does not support. Because it can not execute bash shell script file.